client/lib/date/index.js

```diff
@@ -23,6 +23,16 @@ import { compareValues } from 'components/date-picker/compare-periods';
  * @property {moment.Moment} before - End of the date range.
  */
 
+/**
```

Comment:
Looks like this block might have been pasted here on accident?

Reply:
I was having trouble describing the object in the `@return` line of the jsDoc (line 227), so I created this `@typedef {Object} dateParams` because it gets used a few times.